addons/api/addon/transforms/account-attribute-map-array.js

```diff
@@ -0,0 +1,16 @@
+import Transform from '@ember-data/serializer/transform';
+
+export default class AccountAttributeMapArrayTransform extends Transform {
```

Comment:
Is this literally the same as `AccountClaimMapArrayTransform`? I don't think we should duplicate the exact same transform if so

Reply:
Yea was going to pose that question.  Should we rename and refactor it for use with both?